dask_geopandas/io/parquet.py

```diff
@@ -0,0 +1,126 @@
+from functools import partial
+from typing import TYPE_CHECKING
+
+import pandas as pd
+
+import geopandas
+
+import dask.dataframe as dd
+
+try:
+    # pyarrow is imported here, but is an optional dependency
+    from dask.dataframe.io.parquet.arrow import ArrowEngine
+except ImportError:
+    ArrowEngine = object
+
+if TYPE_CHECKING:
+    import pyarrow
+
+
+class GeoArrowEngine(ArrowEngine):
```

Comment:
Is the idea that this file would possibly contain the fastparquet engine in the future? Should we add a stub to at least suggest people use pyarrow instead in the meantime?

Reply:
I am not planning to add a fastparquet engine myself. To start, geopandas would need to be extended to work with fastparquet as well (we leave the geo metadata parsing to geopandas, and that is currently written with the pyarrow api's)

We should certainly clarify that you can't specify `engine="fastparquet"/"pyarrow"` at the moment, because that will select dask.dataframe's backend, which will fail for geodataframes.

Reply:
Oh yeah I don't think we need to actually write a fastparquet engine. I missed the fixed engine part. I think it makes sense to raise an error if the user tries to pass an engine.